front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -22,6 +25,35 @@ const ScenarioDescription = ({
 }: ScenarioDescriptionProps) => {
   const { t } = useTranslation('operationalStudies/scenario');
   const { openModal } = useModal();
+  const [isOpenedDescription, setIsOpenedDescription] = useState<boolean>(false);
+  const descriptionRef = useRef<HTMLDivElement | null>(null);
+  const ref = useRef<HTMLDivElement | null>(null);
+  const [isTooLongDescription, setIsTooLongDescription] = useState<boolean>(false);
+
+  const toggleDescription = () => {
+    setIsOpenedDescription(!isOpenedDescription);
+  };
+
+  const checkIfTooLong = () => {
+    if (ref && ref.current) {
+      ref.current.scrollHeight > 60
```

Comment:
Not a fan of hardcoding a value in pixels here. Maybe could use `clientHeight` instead?

https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollHeight#determine_if_the_content_of_an_element_is_overflowing

Reply:
Done